ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -313,8 +382,33 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-mag-ipi
+- as: azure-mag-ipi-fullyprivate-f28
+  cron: 57 8 3 * *
+  steps:
+    cluster_profile: azuremag-qe
+    env:
+      BASE_DOMAIN: qemag.azure.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-mag-ipi-fullyprivate
+- as: azure-stack-ipi-proxy-f28
+  cluster: build01
+  cron: 30 13 4 * *
+  steps:
+    cluster_profile: azurestack-qe
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-stack-ipi-proxy
+- as: azure-stack-upi-f28
```

Comment:
This install profile was introduced since 4.12, right?

Reply:
There is indeed no `azure-stack` profiles before 4.12, but I don't know if they are introduced from 4.12.

Reply:
Let use follow a simple rule, for this y-stream upgrade ci, its source version is 4.11, so check the 4.11 e2e ci jobs, if the install profile is existing there, then we run the y-stream upgrade ci for the install profile.